db/flags.sql

```diff
@@ -7,9 +7,21 @@ CREATE TABLE IF NOT EXISTS flags.kvdata (
     corrected REAL NULL,
     controlinfo TEXT NULL,
     useinfo TEXT NULL,
+    -- TODO: check that this type is correct, it's stored as a string in Kvalobs?
     cfailed INT4 NULL,
     CONSTRAINT unique_kvdata_timeseries_obstime UNIQUE (timeseries, obstime)
 );
 
 CREATE INDEX IF NOT EXISTS kvdata_obtime_index ON flags.kvdata (obstime);
 CREATE INDEX IF NOT EXISTS kvdata_timeseries_index ON flags.kvdata USING HASH (timeseries); 
+
+CREATE TABLE IF NOT EXISTS flags.kdvh (
+    timeseries INT4 REFERENCES public.timeseries,
+    obstime TIMESTAMPTZ NOT NULL,
+    controlinfo TEXT NULL,
```

Comment:
This is where the reverse-engineered flags would be stored? Or the original 5 digits of useinfo that KDVH has kept?

Reply:
Both, the `controlinfo` is completely reverse-engineered, while the 5-digit KDVH flags are stored in `useinfo` with the 11 trailing digits set to their default values (line 242 in `kdvh/import_functions.go`). They should both be following what's defined in the excel document you shared with me.